src/main/java/org/gaul/s3proxy/S3ProxyHandler.java

```diff
@@ -2029,6 +2035,18 @@ private void handlePutBlob(HttpServletRequest request,
 
         response.addHeader(HttpHeaders.ETAG, maybeQuoteETag(eTag));
     }
+    private void handleVersionRequest(HttpServletResponse response) throws IOException {
+        response.setStatus(HttpServletResponse.SC_OK);
+        response.setContentType("application/json");
+        response.setCharacterEncoding("UTF-8");
+    
+        String versionInfo = "{ \"status\": \"OK\"}";
```

Comment:
Extra space between `{` and `\"`.

Reply:
Maybe this should return the current time, the time s3proxy launched, and the git hash?  Are there any well-known statusz formats this could follow?